plenum/test/view_change/test_view_change_no_quorum.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from plenum.test.view_change.helper import ensure_view_change
+from plenum.test.helper import stopNodes
+
+
+def test_view_change_no_quorum(nodeSet, up, looper):
+    """
```

Comment:
They cannot be easily merged into one because they require different setup (number of active nodes).
It is quite okay to have two tests for this - one tests upper bound, another one lower bound.